src/main/scala/fr/davit/capturl/scaladsl/Iri.scala

```diff
@@ -309,20 +342,11 @@ final case class LazyIri(
 
   override def toString: String = {
     val b = new StringBuilder()
-    rawScheme.foreach(s => b.append(s"${schemeResult.getOrElse(s)}://"))
-    rawNormalizedAuthority.foreach(b.append)
-    rawNormalizedPath.foreach(b.append)
+    rawScheme.foreach(s => b.append(s"${schemeResult.getOrElse(s)}:"))
+    rawAuthority.foreach(a => b.append(s"${normalizedAuthorityResult.getOrElse(a)}:"))
+    rawPath.foreach(p => b.append(normalizedPathResult.getOrElse(p)))
```

Comment:
if raw path is empty, normalized path can be /. This wont be appended in the string builder